rpc/jsonrpc/types/methods.go

```diff
@@ -899,6 +899,19 @@ func NewRenameAccountCmd(oldAccount, newAccount string) *RenameAccountCmd {
 	}
 }
 
+// RemoveScriptCmd defines the removescript JSON-RPC command.
+type RemoveScriptCmd struct {
+	Script string
+}
+
+// NewRemoveScriptCmd returns a new instance which can be used to issue a
+// removescript JSON-RPC command.
+func NewRemoveScriptCmd(script string) *RemoveScriptCmd {
```

Comment:
Going forward, I would prefer to not add these constructor functions to the types package.  It causes extra/redundant calls or breaking changes to add new fields, when the consumer could simply construct the struct with the named fields instead.